client/extensions/woocommerce/app/products/product-form.js

```diff
@@ -47,19 +47,21 @@ export default class ProductForm extends Component {
 			}
 		);
 		return (
-			<FoldableCard
-				icon=""
-				expanded={ true }
-				className="product-variations"
-				header={ ( <FormToggle onChange={ this.handleToggle } checked={ this.state.isVariation }>
-				{variationToggleDescription}
-				</FormToggle>
-				) }
-			>
-				{ this.state.isVariation && (
-					<ProductVariationTypesForm />
-				) }
-			</FoldableCard>
+			<div className="woocommerce products__form">
```

Comment:
Should we consider putting the top-level "woocommerce" div at the top-level for our page? So that way we don't have to keep adding it to every other component like this as we go?